capitalizedecorator.rb

```diff
@@ -0,0 +1,12 @@
+require 'BaseDecorator'

```

Comment:
* Kindly import the right file properly as the `./basedecorator.rb` is in lowercase